staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1691,6 +1694,38 @@ type CSIPersistentVolumeSource struct {
 	NodePublishSecretRef *SecretReference `json:"nodePublishSecretRef,omitempty" protobuf:"bytes,8,opt,name=nodePublishSecretRef"`
 }
 
+// Represents a source location of a volume to mount, managed by an external CSI driver
+type CSIVolumeSource struct {
+	// Driver is the name of the driver to use for this volume.
```

Comment:
Extend these comments to explain that the name of the driver is part of the CSI standard and is the name the driver knows itself as.  Somethign like that

Reply:
Got it. Will do.